include/oneapi/dpl/pstl/numeric_impl.h

```diff
@@ -194,7 +194,8 @@ __brick_transform_scan(_RandomAccessIterator __first, _RandomAccessIterator __la
 template <typename _Tp, typename _BinaryOperation>
 using is_arithmetic_udop =
     ::std::integral_constant<bool, ::std::is_arithmetic<_Tp>::value &&
-                                       !::std::is_same<_BinaryOperation, ::std::plus<_Tp>>::value>;
+                                       !::std::is_same<_BinaryOperation, ::std::plus<_Tp>>::value &&
```

Comment:
I notice in some places you are using `is_same` and in others `is_same_v`. Could they all be `is_same_v`?

Reply:
@adamfidel My proposal to ignore this moment at current PR.
I have been prepared separate PR: https://github.com/oneapi-src/oneDPL/pull/1107
May be we apply this PR later.